components/eamxx/src/physics/rrtmgp/eamxx_rrtmgp_process_interface.cpp

```diff
@@ -1308,12 +1293,12 @@ void RRTMGPRadiation::run_impl (const double dt) {
 
       // Compute layer cloud mass (per unit area)
 #ifdef RRTMGP_ENABLE_YAKL
-      scream::rrtmgp::mixing_ratio_to_cloud_mass(qc, cldfrac_tot, p_del, lwp);
-      scream::rrtmgp::mixing_ratio_to_cloud_mass(qi, cldfrac_tot, p_del, iwp);
+      rrtmgp::mixing_ratio_to_cloud_mass(qc, cldfrac_tot, p_del, lwp);
+      rrtmgp::mixing_ratio_to_cloud_mass(qi, cldfrac_tot, p_del, iwp);
 #endif
 #ifdef RRTMGP_ENABLE_KOKKOS
-      scream::rrtmgp::mixing_ratio_to_cloud_mass(qc_k, cldfrac_tot_k, p_del_k, lwp_k);
-      scream::rrtmgp::mixing_ratio_to_cloud_mass(qi_k, cldfrac_tot_k, p_del_k, iwp_k);
+      interface_t::mixing_ratio_to_cloud_mass(qc_k, cldfrac_tot_k, p_del_k, lwp_k);
+      interface_t::mixing_ratio_to_cloud_mass(qi_k, cldfrac_tot_k, p_del_k, iwp_k);
       COMPARE_ALL_WRAP(std::vector<real2d>({lwp, iwp}),
```

Comment:
Do we need to run all these checks every time? There are ~10 of these in this file. This seems like a unit-test kind of check. At the very least, can we disable these checks in RELEASE builds? I looked at the rrtmgp submodule, and it seems like this check is always active..

Reply:
By check, do you mean COMPARE_ALL? That only does something in a very unusual configuration (both YAKL and Kokkos are on) in which performance would already be awful since we doing everything twice.